magento/pwa-studio
# [Docs] update routing tutorial

## Description

Updates the routing tutorial to use the extensibility framework.

## Related Issue





Closes PWA-795
Closes #2623 

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs project: `cd pwa-devdocs`
2. Start the HTML preview server: `yarn develop`
3. Navigate to `/tutorials/pwa-studio-fundamentals/add-a-static-route/`
4. Verify new content is accurate.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
